frontend/components/ActivityDetails/InstallDetails/AppInstallDetails/AppInstallDetails.tsx

```diff
@@ -84,43 +104,71 @@ export const getStatusMessage = ({
   if (displayStatus === "failed_install") {
     return (
       <>
-        The MDM command (request) to install <b>{software_title}</b> on{" "}
-        {formattedHost} failed. Please re-attempt this installation.
+        The MDM command (request) to install <b>{appName}</b>
+        {!isDUP && <> on {formattedHost}</>} failed
+        {!isDUP && displayTimeStamp && <> {displayTimeStamp}</>}. Please
+        re-attempt this installation.
       </>
     );
   }
 
+  const renderSuffix = () => {
+    if (isDUP) {
+      return null;
+    }
+    return (
+      <>
+        {" "}
+        on {formattedHost}
+        {displayStatus === "pending_install" && " when it comes online"}
+        {displayTimeStamp && <> {displayTimeStamp}</>}
+      </>
+    );
+  };
   // Create predicate and subordinate for other statuses
   return (
     <>
-      Fleet {getInstallDetailsStatusPredicate(displayStatus)}{" "}
-      <b>{software_title}</b> on {formattedHost}
-      {displayStatus === "pending" ? " when it comes online" : ""}.
+      Fleet {getInstallDetailsStatusPredicate(displayStatus)} <b>{appName}</b>
+      {renderSuffix()}.
     </>
   );
 };
 
 const baseClass = "app-install-details";
```

Comment:
Naming suggestion: rename to `app-install-details-modal` to match `software-install-details-modal`, same for file and directory name to match other file/directory naming conventions

Reply:
Agree. Now that I'm out of the weeds a bit I'll go ahead and do this.